linkerd/meshtls/rustls/src/server.rs

```diff
@@ -130,18 +130,15 @@ fn client_identity<I>(tls: &tokio_rustls::server::TlsStream<I>) -> Option<Client
     let certs = session.peer_certificates()?;
     let c = certs.first().map(Certificate::as_ref)?;
     let end_cert = webpki::EndEntityCert::try_from(c).ok()?;
-    let dns_names = end_cert.dns_names().ok()?;
+    let mut dns_names = end_cert.dns_names().ok()?;
```

Comment:
hmm, looks like the current CI failure is due to `rustls` now returning a `BadDer` error from the call to `end_cert.dns_names()`...trying to figure out what's up there.

Reply:
This was caused by rustls/webpki#167, and fixed by rustls/webpki#169.